vtysh/vtysh.c

```diff
@@ -3939,6 +3939,12 @@ DEFUN (vtysh_ping,
 	return CMD_SUCCESS;
 }
 
+DEFUN(vtysh_motd, vtysh_motd_cmd, "motd", "Show motd")
```

Comment:
"Show motd\n" please.  Also let's update the documentation as well.

Reply:
Added \n.
I did not find where and what can be changed in the documentation.
When you press "?" the description appears in the vtysh console.